MIGRATION.md

```diff
@@ -83,13 +83,13 @@ The markup has been rearranged, and some classes have been removed.
 			</div>
 -			<div class="o-subs-card__copy-pitch">Access to FT.com on your desktop, mobile and tablet</div>
 				<div class="o-subs-card__expander">
--		 		<div class='o-subs-card__read-more'>Read more</div>
+-		 		<div class='o-subs-card__read-more'></div>
 					<div class="o-subs-card__copy-details">
 						<ul class="o-subs-card__copy-benefits">
 							<li>...</li>
 						</ul>
 					</div>
-+					<div class='o-subs-card__read-more'>Read more</div>
++					<button class='o-subs-card__read-more'></button>
```

Comment:
Markup changes are considered breaking changes for Origami components, as we don't provide templates. To make this change we'll need to release a new major (v6.0.0) and add a new migration guide - the rest of the migration guide, for older versions, shouldn't be changed.

Reply:
Yeah, got it.  I agree it's a breaking change and hence the major version upgrade makes complete sense.

Is there a process I can follow for releasing it to the next major version?

Reply:
We don't have guides for creating a new major release - we should write some! 

A major version will be released automatically when this PR is approved and merged, because we've added the Githib label `release:major` (a Github action will see that and make the release for us). Conveniently I've just released a major of `o-topper` if you would like to look at that for an example. Note the update to the README table and MIGRATION guide. https://github.com/Financial-Times/o-topper/pull/94